packages/web-components/src/anchor-button/anchor-button.template.ts

```diff
@@ -8,11 +8,7 @@ import type { AnchorButton, AnchorOptions } from './anchor-button.js';
  */
 export function anchorTemplate<T extends AnchorButton>(options: AnchorOptions = {}): ViewTemplate<T> {
   return html<T>`
-    <template
-      tabindex="0"
-      @click="${(x, c) => x.clickHandler(c.event as PointerEvent)}"
-      @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
-    >
+    <template tabindex="0" @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}">
```

Comment:
Link currently extends the `BaseAnchor` class, so we likely need to make a small change to the template there. I think the build is likely breaking due to the click handler having been removed from the base class.